### PR TITLE
python script

### PR DESCRIPTION
**Pull request by jburel**
* review the setup.rst to match other repo
* add simple_frap.py. This is the script matching the notebook
* add a gettingstarted.rst with code snippets

Follow the instructions on how to set up a conda env
then run from ``python simple_frap.py``

cc @pwalczysko

### CODE REVIEW

File: docs/gettingstarted.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+Getting started with the OMERO Python API
+ ========================================
```
Comment (pwalczysko) [inline docs/gettingstarted.rst]: This header does not build well.
Maybe the underscore is not fully to the left as it should be ?
Screen Shot 2020-05-01 at 17 23 03

File: docs/gettingstarted.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+We will use a Python script showing how to analyze data stored in an OMERO server.
+
+We will show:
+
+- How to connect to server.
+
+- How load images from a dataset.
+
+- Run a simple FRAP analysis measuring the intensity in a named Channel within an existing ellipse.
```
Comment (pwalczysko) [inline docs/gettingstarted.rst]:
```suggestion
- Run a simple FRAP analysis measuring the intensity within a pre-existing ellipse ROI in a named Channel.
```

File: docs/gettingstarted.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+- How to save the generated mean intensities and linked them to the image(s).
```
Comment (pwalczysko) [inline docs/gettingstarted.rst]:
```suggestion
- How to save the generated mean intensities and link them to the image(s).
```

File: docs/gettingstarted.rst [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+**Setup**
+---------
+
+We recommend to use a Conda environment to install the OMERO Python bindings. Please read first :doc:`setup`.
+
+
+**Step-by-Step**
+----------------
+
+In this section, we go over the various steps required to analyse the data.
```
Comment (pwalczysko) [inline docs/gettingstarted.rst]:
```suggestion
In this section, we go through the steps required to analyse the data.
```

File: docs/gettingstarted.rst [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+**Setup**
+---------
+
+We recommend to use a Conda environment to install the OMERO Python bindings. Please read first :doc:`setup`.
+
+
+**Step-by-Step**
+----------------
+
+In this section, we go through the steps required to analyse the data.
```
Comment (mtbc) [inline docs/gettingstarted.rst]:
```suggestion
In this section, we go through the steps required to analyze the data.
```
Was "analyze" above.

File: docs/setup.rst [hunk 6]
```diff
@@ -33,3 +38,32 @@ Conda manages programming environments in a manner similar to
...
+
+- Select ``Terminal``
+
+.. image:: images/terminal.png
+
+- A Terminal will open in a new Tab
+
+- A Conda environment has already been created when the Docker Image was built
+
+- To list all the Conda environment, run::
```
Comment (mtbc) [inline docs/setup.rst]:
```suggestion
- To list all the Conda environments, run::
```
Maybe ... ? I don't know Conda.

File: docs/setup.rst [hunk 7]
```diff
@@ -33,3 +38,32 @@ Conda manages programming environments in a manner similar to
...
+
+    $ conda env list
+
+- The environment with the OMERO Python bindings and few other libraries is named ``notebook``, activate it::
```
Comment (mtbc) [inline docs/setup.rst]:
```suggestion
- The environment with the OMERO Python bindings and a few other libraries is named ``notebook``, activate it::
```

File: scripts/simple_frap.py [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+# Load-images
+def load_images(conn, dataset_id):
+    dataset = conn.getObject("Dataset", dataset_id)
+    images = []
+    for image in dataset.listChildren():
+        images.append(image)
+    return images
+
+
+# Analyse-images
```
Comment (mtbc) [inline scripts/simple_frap.py]:
```suggestion
# Analyze-images
```
to match the method name :smiley:

**Comment by will-moore** [comment]
This worked for me (gave up on repo2docker as it was taking too long to download/setup env):

```
$ python simple_frap.py 
Host [wss://workshop.openmicroscopy.org/omero-ws]: localhost
Username [trainer-1]: will
Password: 
Dataset ID [2391]: 403
Channel Label [GFP]: 
Processing Image: 1265
ROI: 175180
ROI: 208165
Shape: 208165
Channel Index: 0
SizeT 64
Deleting old annotations [6962]
done
```

**Comment by jburel** [comment]
Thanks @will-moore for testing the script

**Comment by will-moore** [comment]
Might be helpful if https://github.com/ome/omero-guide-python/blob/master/README.md gave you the commands to setup a conda env. I had to look that up (don't do it often enough) and forgot to install pip initially, so then had to search how to do that etc.

I also read "install repo2docker from PyPI" and blindly tried "pip install repo2docker" before I noticed the correct instruction below (I guess I was taking each instruction at a time).

**Comment by pwalczysko** [comment]
repo2docker worked fine for me

```
pwalczysko@2e6d929aaace:~$ conda env list
# conda environments:
#
base                     /srv/conda
notebook              *  /srv/conda/envs/notebook

pwalczysko@2e6d929aaace:~$ conda activate notebook
(notebook) pwalczysko@2e6d929aaace:~$
(notebook) pwalczysko@2e6d929aaace:~$
(notebook) pwalczysko@2e6d929aaace:~$ cd scripts/
(notebook) pwalczysko@2e6d929aaace:~/scripts$ python simple_frap.py
Host [wss://workshop.openmicroscopy.org/omero-ws]:
Username [trainer-1]:
Password:
Dataset ID [2391]:
Channel Label [GFP]:
Processing Image: 28662
ROI: 1251954
Shape: 2067304
SizeT 64
Deleting old annotations [131508]
done
(notebook) pwalczysko@2e6d929aaace:~/scripts$
```

**Comment by pwalczysko** [comment]
The text edit fixes also look good.

**Comment by pwalczysko** [comment]
What I am missing though is the info about that when I went through the repo2docker install and successfully am inside that terminal, then the ``simple_frap.py`` script is there for me to use. I just knew that by heart/experience - there is no way to figure that out  from the walkthroughs. It would be enough to put in a sentence "the script ... is there for you in the scripts folder"  in the appropriate place, probably just after the line https://github.com/ome/omero-guide-python/pull/4/files#diff-51acb46255e0479c368e9dc244c9e0c4R69

**Comment by jburel** [comment]
instructions are in ``setup.rst``
I will add a link to it in the README

**Comment by jburel** [comment]
Merging so we can move forward with some of the zarr related work

**Pull request merged**